Makefile

```diff
@@ -0,0 +1,16 @@
+all:
+	cargo build
+
+run:
+	cargo run
+
+test:
+	cargo test -- --nocapture
+
+check:
+	cargo check
+
+lint:
+	cargo clippy -- -D warnings
+
+.PHONY: all run test check lint
```

Comment:
This file is missing a blank line at the end; you should configure your editor to add it automatically.

Reply:
Will do it in next PR